src/adhocracy_core/adhocracy_core/sheets/test_workflow.py

```diff
@@ -52,6 +52,22 @@ def test_create(self, inst, kw, mocker):
                                       ('', 'No workflow'),
                                       ]
 
+    def test_create_standart_in_alternatives(self, inst, kw, mocker,
+                                             resource_meta):
+        from adhocracy_core.schema import SingleLine
+        kw['creating'] = resource_meta._replace(
+            alternative_workflows=('default',))
+        permission_check = mocker.patch('adhocracy_core.sheets.workflow.'
+                                        'create_deferred_permission_validator').return_value
+        inst = inst.bind(**kw)
+        assert isinstance(inst, SingleLine)
+        assert inst.validator.validators[0].choices == ('',
+                                                        'default')
+        assert inst.validator.validators[1] == permission_check.return_value
+        assert inst.widget.values == [('default', 'default'),
+                                      ('', 'No workflow'),
+                                      ]
+
```

Comment:
We want to test widget and validator[0] choices here.
 We don`t need to test again in addition isinstance, validators[1]